### ISSUE TITLE
Subclass an existing provider to extend it or create a new provider

### ISSUE THREAD

**Issue by gnud** [WindSekirun/RxSocialLogin#53]
What's the procedure to register a new provider for instance custom JWT token authentication to own API server?

**Comment by WindSekirun** [comment]
I don't understand your question. 

1. Which services provide custom JWT token authentication?
2. Basically, RxSocialLogin and each providers doesn't support existing provider. You can write new providers with copy-pasting by MR

**Comment by gnud** [comment]
My question was guide to write new providers?
Without forking the project that is, by using the library as it is :).

**Comment by WindSekirun** [comment]
@gnud No. Until now, forking the project is only way to write new providers.
